src/markov/markov_approx.jl

```diff
@@ -13,6 +13,7 @@ https://lectures.quantecon.org/jl/finite_markov.html
 
 import Optim
 import NLopt
+import FastGaussQuadrature: gausshermite
```

Comment:
We should change this to `using FastGaussQuadrature: gausshermite`. Import has two uses:

1. Bring a module name into scope (as we do with Optim and NLopt above)
2. Bring a specific function from a different module into scope _so we can extend the function with our own methods_. We don't add methods to `gausshermite`, so we should just call `using` instead of `import`.